nisystemlink/clients/core/_http_configuration_manager.py

```diff
@@ -23,6 +23,9 @@ class HttpConfigurationManager:
     HTTP_LOCALHOST_CONFIGURATION_ID = "SYSTEMLINK_LOCALHOST"
     """The default ID of the SystemLink Server's configuration on the SystemLink Server itself."""
 
+    JUPYTER_HUB_CONFIGURATION_ID = "SYSTEMLINK_JUPYTER_HUB"
```

Comment:
Since this is a "virtual" configuration that doesn't actually exist on disk, I wonder if we should track it separately than in the `_configs` dict. If someone made a custom configuration with this ID (i.e., created a file for it in `C:\ProgramData\National Instruments\Skyline\HttpConfigurations`), it'll overwrite the Juypter one _and_ get treated as a fallback, which custom configurations don't normally do.

Though it's probably reasonable to treat any configurations starting with `SYSTEMLINK_` as "reserved" and not something customers should create themselves, it could be nice to avoid this ambiguity.

For example, we could have a separate `_juypter_config` attribute next to `_configs`.

Reply:
It makes sense that `id` based configurations are limited to physical configurations read from disk and Jupyter config is special.